crates/simulate/src/contract/sim_contract.rs

```diff
@@ -1,115 +0,0 @@
-/// NOTICE: Please check the `crates/simulate/src/contract/NOTICE` for licensing-related disclosures
```

Comment:
No longer need this wrapper to interact with the evm, since you can send directly use the rust bindings to generate an ethers `ContractCall`.